modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,252 @@
+  resources:
+    - |
+      apiVersion: networking.k8s.io/v1
+      kind: Ingress
+      metadata:
+        name: {{ .ResourceName }}
+        namespace: {{ .Namespace }}
+        labels:
+        annotations:
+      spec:
+       ingressClassName: nginx 
+       tls:
+       - hosts:
+         - {{ .Host }}
+         secretName: testsecret-tls
+       rules:
+       - host: {{ .Host }} 
+         http:
+            paths: 
+            - pathType: Prefix
+              path: {{ .Path }}
+              backend: 
+                service:
+                  name: {{ .Service.Name }}
+                  port:
+                    name: {{ .Service.PortName }}
+----
+
+[calloutlist]
+
+.. Hosts or domains used for mapping the public endpoints configured on the Mule app deployment.
+.. One or more Kubernetes routing resources, such as an Ingress resource.
+.. The Runtime Fabric-generated unique routing resource name. Use this placeholder to make sure the Kubernetes resource name meets the uniqueness constraint. If you don’t use this placeholder, make sure the resource name is unique.
+.. Host or domain from the Mule app's public URL.
+.. Path from the Mule app's public URL. 
+
+
+=== Example for OpenShift Routes
+This example shows an HTTP route-template resource for OpenShift routes:
+
+----
+apiVersion: rtf.mulesoft.com/v1
+kind: HTTPRouteTemplate
+metadata:
+  name: test-template-routes
+  namespace: rtf
+spec:
+  baseEndpoints:
+    - https://demo.example.com
+  resources:
+    - |
+apiVersion: v1
+kind: Route
+metadata:
+  name: {{ .ResourceName }}
+  namespace: {{ .Namespace }}
+spec:
+  host: {{ .Host }}
+  path: {{ .Path }}
+  to:
+    kind: Service
+    name: {{ .Service.Name }}
+  tls:
+    termination: edge
+    key: |-
+      -----BEGIN PRIVATE KEY-----
+      [...]
+      -----END PRIVATE KEY-----
+    certificate: |-              
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+    caCertificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+
+----
+
+=== Example for Kubernetes Gateway API
+
+This example shows an HTTP route-template resource for Kubernetes Gateway API (HTTPRoute):
+
+----
+apiVersion: rtf.mulesoft.com/v1
+kind: HttpRouteTemplate
+metadata:
+  name: k8s-gateway-api-cr
+  namespace: rtf
+spec:
+  baseEndpoints:
+    - https://demo.example.com
+  resources: |-
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: {{ .ResourceName }}
+  namespace: {{ .Namespace }}
+spec:
+  parentRefs:
+  - kind: Gateway
+    name: cluster-gateway
+  hostnames:
+  - {{ .Host }}
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: {{ .Path }}
+    backendRefs:
+    - name: {{ .Service.Name }}
+      port: {{ .Service.Port }}
+
+----
+
+[NOTE]
+You are responsible for the configuration of the Kuberneates Gateway resources and allowing HTTP Routes from the route's namespace to be attached.
```

Comment:
"Kuberneates"

Reply:
"HTTP Routes" -> "HTTPRoutes". Can we provide a link to how to allow routes namespace to be attached?